tests/test_modeling_reformer.py

```diff
@@ -976,3 +981,15 @@ def test_pretrained_generate_crime_and_punish(self):
             output_text,
             "A few months later state expression in his ideas, at the first entrance. He was positively for an inst",
         )
+
+    @slow
+    def test_inference_classification_head(self):
```

Comment:
This makes only sense if the model in question is a sequence classification model and `google/reformer-enwik8` is not. Would be nice to just add one "normal" (non-slow) test as is done in Bert.